### ISSUE TITLE
False positive in `condition_message_linter()` for `list$stop()`

### ISSUE THREAD

**Issue by MichaelChirico** [r-lib/lintr#1455]
As seen in r-devel:

https://github.com/r-devel/r-svn/blob/9bcb4bef2979e608804fb37ab6471b4894bfa2b2/src/library/compiler/R/cmp.R#L2204-L2206

```r
cntxt$stop(
    paste(sQuote(name), "is not a registered math1 function"),
    cntxt, loc = cb$savecurloc())
```

We should simply ignore calls after `$`.
